spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/SparkUtils.java

```diff
@@ -84,7 +85,7 @@ static void validateConfigParameters(Schema inputSchema, @Nullable String featur
       throw new IllegalArgumentException("Cannot specify values for both featuresToInclude and featuresToExclude. " +
                                            "Please specify fields for one.");
     }
-    Map<String, Integer> fields = new HashMap<>();
+    Map<String, Integer> fields = new LinkedHashMap<>();
```

Comment:
There are 4 places where `SparkUtils#getFeatureList()` method is used:

- `DecisionTreePredictor#transform()`
- `DecisionTreeTrainer#run()`
- `SparkUtils#getCategoricalFeatureInfo()`
- `SparkUtils#validateConfigParameters()`

Wondering in which method do we need ordering in any of the above usages?